apps/og/src/pages/api/post/[id].tsx

```diff
@@ -0,0 +1,83 @@
+import type { NextApiHandler } from 'next';
+
+import { PublicationDocument } from '@hey/lens';
+import { apolloClient } from '@hey/lens/apollo';
+import { unstable_createNodejsStream } from '@vercel/og';
+
+const handler: NextApiHandler = async (req, res) => {
+  const POST_ID = req.query.id;
```

Comment:
I guess using postID instead of POST_ID will be cool, cuz its not a constant value